galeracluster/source/training/tutorials/galera-backup.rst

```diff
@@ -205,7 +205,7 @@ To attach a server to a Galera node, so that it may act as a slave, we'll need t
 
 The only other parameter worth mentioning is the ``read-only`` option to make sure no one will edit the data.
 
-Once we've configured the slave, we'll need to restart it.  Then we have to load the data back-up dump file we made on the master, using the ``mysql`` client. And we'll have to execute the ``CHANGE MASTER TO`` (for MySQL releases before 8.4) or ``CHANGE REPLICATION SOURCE TO`` (for MySQL releases 8.4 und upper) statement on the slave to tell it who is the master:
+Once we've configured the slave, we'll need to restart it.  Then we have to load the data back-up dump file we made on the master, using the ``mysql`` client. And we'll have to execute the ``CHANGE MASTER TO`` (MySQL < 8.4) or ``CHANGE REPLICATION SOURCE TO`` (MySQL > 8.4) statement on the slave to tell it who is the master:
```

Comment:
as mentioned before, the text includes things like master/slave which we also have to update